### ISSUE TITLE
upload pretrained weights

### ISSUE THREAD

**Issue by pengzhangzhi** [david3684/flm#2]
Hi good work I wonder whats the training cost, like how long and how many gpus and would you release the pretrained weights?

**Comment by david3684** [comment]
Hi, thanks for interest in our work. We used 8xH100s for OWT Training. We're working on final touches with the experiments and would upload the checkpoints very soon. Thanks!

**Comment by david3684** [comment]
Checkpoints are available now! Please refer to the updated README.

**Comment by pengzhangzhi** [comment]
Hey thanks! what is the training cost on the two datasets?

**Comment by david3684** [comment]
FLM training takes about 2 days for LM1B and 5 days for OpenWebText, with 8xH100s and torch.compile (without compiling you might twice the gradient accumulation for OpenWebText training). For distillation you could finish both within a day.
